packages/cli/src/utils/preprocess.ts

```diff
@@ -7,21 +7,16 @@ import { readFileSync } from 'fs'
 import nodemailer from 'nodemailer'
 
 export class FunctionPreProcess {
-  constructor(private isDev: boolean = false) {}
   match(step: Task): Promise<boolean> {
     return step.type === 'task' && step.handler
   }
 
   async replace(step: Task): Promise<Task> {
-    const instance: Instance = this.isDev
-      ? {
-          instance: {
-            src: 'https://github.com/liteflow-services/js-function/',
-          },
-        }
-      : { instanceId: '26842e9b-86e3-42e6-9b66-e890006c7b2d' } // Current deployed version of service js-function
     return {
-      ...instance,
+      instance: {
+        src:
+          'https://github.com/liteflow-services/js-function/archive/v1.tar.gz',
```

Comment:
Why not using the tag? It seems to work correctly with git and don't need the #252 

```suggestion
          'https://github.com/liteflow-services/js-function#v1',
```

Reply:
I feel tarball is much simpler, faster and lighter.
The git system is doing way more stuff and may have side effect on other platform.